app/assets/stylesheets/variables/_vendor.scss

```diff
@@ -2,3 +2,16 @@ $theme-body-font-size: 'sm';
 $theme-font-path: 'identity-style-guide/dist/assets/fonts';
 $theme-image-path: 'identity-style-guide/dist/assets/img';
 $theme-global-border-box-sizing: true;
+$theme-grid-container-max-width: 'tablet-lg';
+$theme-header-min-width: 'tablet';
+$theme-utility-breakpoints: (
+  'card': false,
+  'card-lg': false,
+  'mobile': false,
+  'mobile-lg': true,
+  'tablet': true,
+  'tablet-lg': false,
+  'desktop': false,
```

Comment:
Disabling this will have a nice performance impact, reducing the size of the generated stylesheet by a fair bit, since each of these breakpoints generates a lot of utility styles.

Overall, this pull request reduces the size of the stylesheet by ~11% (62.2kb to 55.4kb gzipped).